v22.2/create-sequence.md

```diff
@@ -31,17 +31,17 @@ The user must have the `CREATE` [privilege](security-reference/authorization.htm
  Parameter | Description
 -----------|------------
 `seq_name` | The name of the sequence to be created, which must be unique within its database and follow the [identifier rules](keywords-and-identifiers.html#identifiers). When the parent database is not set as the default, the name must be formatted as `database.seq_name`.
-`INCREMENT` | The value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.<br><br>**Default:** `1`
-`MINVALUE` | The minimum value of the sequence. Default values apply if not specified or if you enter `NO MINVALUE`.<br><br>**Default for ascending:** `1` <br><br>**Default for descending:** `MININT`
-`MAXVALUE` | The maximum value of the sequence. Default values apply if not specified or if you enter `NO MAXVALUE`.<br><br>**Default for ascending:** `MAXINT` <br><br>**Default for descending:** `-1`
-`START [WITH]` | The first value of the sequence. <br><br>**Default for ascending:** `1` <br><br>**Default for descending:** `-1`
+`INCREMENT` | The value by which the sequence is incremented. A negative number creates a descending sequence. A positive number creates an ascending sequence.<br><br>**Default:** `1`.
+`MINVALUE` | The minimum value of the sequence. Default values apply if not specified or if you enter `NO MINVALUE`.<br><br>**Default for ascending:** `1` <br><br>**Default for descending:** `MININT`.
```

Comment:
In my opinion, adding punctuation after a default value specification is not necessary, and creates visual noise. I think that a user will expect to see only the value itself following the text `**Default:**`. Our [punctuation rules](https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#punctuation-rules) don't require this, and if necessary I can update the style guide to explicitly advise against it. However, this has not been the prevailing usage on our docs so I think that it can also be implicit.

<img width="261" alt="image" src="https://user-images.githubusercontent.com/8740013/191356899-a017f551-79ce-4057-9456-01e0a9e561d1.png">

Reply:
Removed.

I added them because normally when you have some paragraphs ending in a . all should end in the same way.